Makefile

```diff
@@ -106,3 +111,9 @@ setup-kind:
 .PHONY: install-helm
 install-helm:
 	curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
+
+.PHONY: e2e-local-bootstrap
+e2e-local-bootstrap:
+	kind create cluster --image kindest/node:v${KIND_K8S_VERSION}
```

Comment:
Can we reuse existing rules which create the kind cluster and build image?

Reply:
since the binaries for the tooling are downloaded throughout multiple make targets and also being dependent on conditions, I found it much cleaner to have a separate target that can set up the local environment

Otherwise, `e2e-bootstrap, setup-kind, e2e-container` would all need to be rewritten and possibly have more conditionals inside that would make it confusing to follow (such as an additional condition flag that will ask if local, then skip binary download, for installing helm, kubectl and kind which are throughout different targets)